FileUploader: describe abortUpload (#1264)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxFileUploader/3 Methods/abortUpload(fileIndex).md

---
---
##### shortDescription
Cancels the file upload.

##### param(fileIndex): Number

> The index of a file whose upload is being cancelled. 

Reviewer: ```suggestion
The index of the file for which the upload is cancelled.
```

---

---

##### jQuery

    <!-- tab: JavaScript -->
    var uploadControl = $("#uploaderContainer").dxFileUploader("instance");

    uploadControl.abortUpload(1)

---

[note]

> The **abortUpload** method works differently in **useForms**, **useButtons** and **instantly** [upload modes](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode):

Reviewer: ```suggestion
The **abortUpload** method works differently in the following [upload modes](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode):
```

- **useForms**: The method is not supported in this mode.

> - **useButtons**: Cancels the file upload and makes the file ready for uploading.  

Reviewer: See my similar comment above.

Author: It is "available for upload"

Reviewer: ```suggestion
- **useButtons**: Cancels the file upload and makes the file available for upload.  
```
